pkg/coscheduling/core/core.go

```diff
@@ -112,7 +119,37 @@ func (pgMgr *PodGroupManager) PreFilter(ctx context.Context, pod *corev1.Pod) er
 		return fmt.Errorf("cannot found engough pods, "+
 			"current pods number: %v, minMember of group: %v", len(pods), pg.Spec.MinMember)
 	}
-	return nil
+
+	// TODO(cwdsuzhou): This resource check can not absolutely to avoid unschedulable group. We should add Post-Filter to reject all if anyone does not fit.
```

Comment:
This comment sounds a bit ambiguous to me, esp. the PostFilter part.

I think you can just focus on explaining how it works in PreFilter as well as emphasizing its limitation (only PreFilter resource constraints so even if a PodGroup passed here, it may not necessarily pass Filter due to other constraints such as affinity/taints, etc.)